README.md

```diff
@@ -42,6 +42,27 @@ On releases v0.8.4 and above, kubemacpool is set to apply on pods/vms that resid
 - `mutatepods.kubemacpool.io=allocateForAll` - to opt in pods mac allocation in your namespace
 - `mutatevirtualmachines.kubemacpool.io=allocateForAll` - to opt in vms mac allocation in your namespace
 
+#### How to enable/disable kubemacpool for a namespace 
+
+Kubemacpool is disabled by default on a new namespace.
+To enable kubemacpool on a specific namespace:
+```bash
+kubectl label namespace user-namespace-opting-in-pods-vms mutatepods.kubemacpool.io=allocateForAll mutatevirtualmachines.kubemacpool.io=allocateForAll
+namespace/user-namespace-opting-in-pods-vms labeled
+```
+
+To disable kubemacpool in a namespace:
+```bash
+kubectl label namespace user-namespace-opting-in-pods-vms mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
+namespace/user-namespace-opting-in-pods-vms labeled
+```
+
+**note:** if the kubemacpool's mutatingwebhookconfiguration `kubemacpool-mutator` namespace-selector value per vm/pod is set to `allocateForAll`, then you can also opt-out your namespace by setting the label value to `disabled`:
```

Comment:
How would I change the default, would you document it? Won't block this PR with it, but I think it would save tomorrow's us some time

Reply:
You say 'disabled' here, but the command spells 'disable'

Reply:
Now that I think of it, is it even possible for a user to change the mutatingwebhookconfiguration? (on U/S? on D/S?)
or should I address this note for the administrator?

Reply:
It is for the user of this component. Person that deploys KMP directly can change any of that. Maintainer of another component consuming this (e.g. CNAO) would like to know how to do it too.